L/LLVM/common.jl

```diff
@@ -134,6 +134,11 @@ CMAKE_FLAGS+=(-DLLVM_LINK_LLVM_DYLIB:BOOL=ON)
 # set a SONAME suffix for FreeBSD https://github.com/JuliaLang/julia/issues/32462
 CMAKE_FLAGS+=(-DLLVM_VERSION_SUFFIX:STRING="jl")
 
+if [[ "${target}" == *linux* || "${target}" == *mingw* ]]; then
+    # https://bugs.llvm.org/show_bug.cgi?id=48221
+    CMAKE_CXX_FLAGS+="-fno-gnu-unique"
```

Comment:
We might need to add this to `CFLAGS` as well as `CXXFLAGS` if there is any code that is treated as `C` and not `C++`.

Reply:
The primary issue is around static variables and initializers so I "think" C++ only suffices